doc/2/guides/introduction/1-what-is-kuzzle/index.md

```diff
@@ -0,0 +1,52 @@
+---
+code: false
+type: page
+title: What is Kuzzle
+description: Why are we spending so much time developing Kuzzle
+order: 100
+---
+
+# What is Kuzzle
+
+If you're on this page it's probably because you need a backend for your mobile, web or IoT application.
+
+Once again you had been preparing to develop a backend from scratch, not really from scratch because you probably planned to use a framework like Fastify or Nest.js to make it easier for you.  
+This kind of framework allows you to develop faster by providing a predefined structure, classes and configurations.
```

Comment:
If that's possible I find it preferable to not give names.

```suggestion
Once again you had been preparing to develop a backend from scratch... Well, maybe not entirely from scratch, because you probably planned to use some kind of framework (and there are a lot of them!) to make it easier for you.  
Those frameworks allow you to develop faster by providing a predefined structure, classes and configurations.
```